src/test/java/com/puppycrawl/tools/checkstyle/checks/TranslationCheckTest.java

```diff
@@ -120,10 +120,53 @@ public void testLogIOException() throws Exception {
         check.setMessageDispatcher(createChecker(checkConfig));
 
         Method logIOException = check.getClass().getDeclaredMethod("logIOException",
-                IOException.class,
-                File.class);
+            IOException.class,
+            File.class);
         logIOException.setAccessible(true);
         logIOException.invoke(check, new IOException("test exception"), new File(""));
     }
 
+    @Test
+    public void testDefaultTranslationFileIsMissing() throws Exception {
+        final DefaultConfiguration checkConfig = createCheckConfig(TranslationCheck.class);
+        checkConfig.addAttribute("requiredTranslations", "ja, de, ja");
```

Comment:
this test could include more complicated string set like ",ja,de,, ja," just to have full functionality coverage